vaadin-checkbox-flow-parent/vaadin-checkbox-flow/src/test/java/com/vaadin/flow/component/checkbox/tests/CheckboxGroupTest.java

```diff
@@ -118,18 +113,12 @@ public void unsetReadOnlyDisabledCheckboxGroup_groupIsDisabledAndNotReadonly() {
     }
 
     @Test
-    public void setReadOnlyEnabledCheckboxGroup_groupIsDisabledAndNotReadonly() {
+    public void setReadOnlyEnabledCheckboxGroup_groupIsNotDisabled() {
```

Comment:
I feel this test is not needed anymore as it was just testing the previous workaround.

Reply:
Right, thanks. Removed the test.